libraries/botbuilder-dialogs-adaptive-runtime-core/src/configuration.ts

```diff
@@ -26,7 +26,7 @@ export interface Configuration {
  * Useful for shimming BotComponents into ComponentRegistrations
  */
 export const noOpConfiguration: Configuration = {
-    get(_path: string[]): unknown | undefined {
+    get(_path: string[]): undefined {
```

Comment:
You could try to drop these type annotations. Explicitly typing `noOpConfiguration` with `: Configuration` means they should probably be inferred.

Reply:
Good call, this slipped my mind; removed.